src/style.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable color-hex-case */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background: white;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060B35;
+  margin: 0 0 4px;
+  font-style: medium;
```

Comment:
think about it. what is a medium style? something that does not exist i think :)